pod/perlre.pod

```diff
@@ -1193,54 +1257,68 @@ would match the same as C</(Y) ( (X) \g3 \g1 )/x>.  This allows you to
 interpolate regexes into larger regexes and not have to worry about the
 capture groups being renumbered.
 
-You can dispense with numbers altogether and create named capture groups.
-The notation is C<(?E<lt>I<name>E<gt>...)> to declare and C<\g{I<name>}> to
-reference.  (To be compatible with .Net regular expressions, C<\g{I<name>}> may
-also be written as C<\k{I<name>}>, C<\kE<lt>I<name>E<gt>> or C<\k'I<name>'>.)
+As mentioned earlier you can dispense with numbers altogether and create
+named capture groups. The notation is C<(?E<lt>I<name>E<gt>...)> to
+declare and C<\g{I<name>}> to reference. (To be compatible with .Net
+regular expressions, C<\g{I<name>}> may also be written as
+C<\k{I<name>}>, C<\kE<lt>I<name>E<gt>> or C<\k'I<name>'>. )
 I<name> must not begin with a number, nor contain hyphens.
-When different groups within the same pattern have the same name, any reference
-to that name assumes the leftmost defined group.  Named groups count in
-absolute and relative numbering, and so can also be referred to by those
-numbers.
-(It's possible to do things with named capture groups that would otherwise
-require C<(??{})>.)
-
-Capture group contents are dynamically scoped and available to you outside the
-pattern until the end of the enclosing block or until the next successful
-match in the same scope, whichever comes first.
-See L<perlsyn/"Compound Statements"> and
+When different groups within the same pattern have the same name, any
+reference to that name assumes the leftmost defined group. Named groups
+count in absolute and relative numbering, and so can also be referred to
+by those numbers. (It's possible to do things with named capture groups
+that would otherwise require C<(??{})>.)
+
+Inside of a I<new> match you can access the contents of a capture group
+from the most recent previous successful match by using the same
+variables as you would outside of a pattern. Thus in the second match in
+the following example
+
+    "foo" =~ /(foo)/;
+    "barfoobar" =~ /(bar)$1\1/;
+
+the C<$1> refers to content of the first capture group from the
+B<previous> match, and the C<\1> refers to the contents of the first
+capture group of the B<current> match.
+
+Capture group contents are dynamically scoped and available to you
+outside the pattern until the end of the enclosing block or until the
+next successful match in the same scope, whichever comes first. See
+L<perlsyn/"Compound Statements"> and
 L<perlvar/"Scoping Rules of Regex Variables"> for more details.
 
-You can access the contents of a capture group by absolute number (using
-C<"$1"> instead of C<"\g1">, I<etc>); or by name via the C<%+> hash,
-using C<"$+{I<name>}">.
+=head4 Use of curly braces in backreference notation
```

Comment:
I like the idea of splitting this into head4 subheadings (it's pretty sorely needed), but this one isn't sufficient on its own: it would imply that the whole rest of the "Capture groups" section is under this subheading which isn't the case. Suggested other subheadings: "Backreferences", "Named capture groups" (moving the last paragraph about capture group scoping before the section on backreferences), and maybe "Backreference notation ambiguity", "Examples", and "Match variables"? some reorganization might be needed.

Reply:
Ack.

Reply:
I am going to park this one until the rest of the feedback is cleared.